src/tools/moment.js

```diff
@@ -13,7 +13,14 @@ const getToday10 = () => {
   return today;
 };
 
+const get2digit = (num) => {
+  if (num < 10) {
+    return `0${num}`;
+  }
+  return num;
+};
```

Comment:
```suggestion
const min2str = (min) => min.toString().padStart(2, '0')
```

함수 명이랑 함수 내부 수정 제안드립니다.

Reply:
저런 함수가 있는지는 몰랐네요. min 뿐만 아니라 hour에도 쓰이는데 함수 명은 어떻게 하는게 좋을까요?

Reply:
@imYourChoi 전 `time2str` 추천드립니다. 괜찮나요?

Reply:
좋습니다~